packages/frontapp/src/app/components/Button/index.tsx

```diff
@@ -136,6 +136,17 @@ const ButtonContainer = styled.button<IButtonProps>`
     max-width: ${dimensions.xl_2};
   }
 
+  svg path {
+    fill: ${({ transparent, secondary }) =>
+      transparent
+        ? secondary
+          ? colors.dropdown_gray
+          : colors.hover_color
+        : colors.bg_secondary};
+    height: auto;
+    max-width: ${dimensions.xl_2};
+  }
+
```

Comment:
I would move it to declaration above to avoid duplication:
`svg, svg path {
  fill and other styles
}`